Buildings/Fluid/Dehumidifiers/Desiccant/BaseClasses/Performance.mo

```diff
@@ -0,0 +1,301 @@
+     iconTransformation(extent={{-120,32},{-100,52}})));
+  Modelica.Blocks.Interfaces.RealInput X_w_ProEnt(
+     final unit="1")
+    "Humidity ratio of the process air entering the dehumidifier"
+    annotation (Placement(transformation(
+    extent={{-124,-54},{-100,-30}}),iconTransformation(extent={{-120,-50},
+    {-100,-30}})));
+  Modelica.Blocks.Interfaces.RealInput VPro_flow(
+     final unit="m3/s")
+     "Volumetric  flow rate of the process air"
+     annotation (Placement(transformation(
+          extent={{-124,-92},{-100,-68}}),
+          iconTransformation(extent={{-120,-92},{-100,-72}})));
+  Modelica.Blocks.Interfaces.RealInput mPro_flow(
+    final unit="kg/s") "Mass flow rate of the process air"
+                                        annotation (Placement(transformation(
+        extent={{-12,-12},{12,12}},
+        rotation=90,
+        origin={0,-112}),   iconTransformation(
+        extent={{-10,-10},{10,10}},
+        rotation=90,
+        origin={0,-110})));
+  Modelica.Blocks.Interfaces.RealInput TRegEnt(
+    final unit="K")
+    "Temperature of the regeneration air entering the dehumidifier" annotation (
+     Placement(transformation(extent={{-124,-14},{-100,10}}),
+        iconTransformation(extent={{-120,-10},{-100,10}})));
+  Modelica.Blocks.Interfaces.RealInput uSpe(
+    final unit="1",
+    final min=0.3,
+    final max=1)
+    "Wheel speed ratio" annotation (Placement(transformation(
+        extent={{12,-12},{-12,12}},
+        rotation=90,
+        origin={-80,112}), iconTransformation(
+        extent={{10,-10},{-10,10}},
+        rotation=90,
+        origin={-82,110})));
+  Modelica.Blocks.Interfaces.RealOutput TProLea(
+    final unit="K")
+    "Temperature of the process air leaving the dehumidifier" annotation (
+      Placement(transformation(extent={{100,70},{120,90}}), iconTransformation(
+          extent={{100,70},{120,90}})));
+  Modelica.Blocks.Interfaces.RealOutput X_w_ProLea(
+    final unit="1")
+    "Humidity ratio of the process air leaving the dehumidifier"
+     annotation (Placement(transformation(extent={{100,30},{120,50}}),
+     iconTransformation(extent={{100,30},{120,50}})));
+  Modelica.Blocks.Interfaces.RealOutput VReg_flow(
+    final unit="m3/s")
+    "Volumetric flow rate of the regeneration air" annotation (Placement(transformation(
+          extent={{100,-50},{120,-30}}), iconTransformation(extent={{100,-50},{120,
+            -30}})));
+  Modelica.Blocks.Interfaces.RealOutput yQReg(
+    final unit="1")
+    "Regeneration heating output ratio" annotation (Placement(transformation(
+          extent={{100,-90},{120,-70}}), iconTransformation(extent={{100,-90},{120,
+            -70}})));
+protected
+  Real CpReg(final unit="J/kg")
+    "Specific regeneration energy";
+  Real etaSen(final unit="1")
+    "Sensible heat exchange effectiveness";
+  Real etaLat(final unit="1")
+    "Latent heat exchange effectiveness";
+
+equation
+  if onDeh then
```

Comment:
in this section, please ensure the indent in right way.

Reply:
Removed the extra spaces.